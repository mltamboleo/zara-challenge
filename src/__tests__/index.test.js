import '@testing-library/jest-dom'
import { render } from '@testing-library/react'
import CharactersList from '../pages/index.tsx'

jest.mock('next/router', () => ({
  useRouter() {},
}));

describe('Home Page', () => {
  it('Renders Home', () => {
    const { container } = render(<CharactersList />)
    expect(container).toMatchSnapshot()
  })
})
