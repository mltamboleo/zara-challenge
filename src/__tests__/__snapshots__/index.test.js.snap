// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page Renders Home 1`] = `
<div>
  <div
    class="home"
  >
    <div
      class="home__favs-header home__favs-header--hidden"
    >
      <h1
        class="home__favs-header-text"
      >
        FAVORITES
      </h1>
    </div>
    <div
      class="home__search"
    >
      <input
        class="home__search-input"
        placeholder="SEARCH A CHARACTER..."
        type="search"
      />
      <p
        class="home__search-results-label"
      >
        0
         
        results
      </p>
    </div>
    <div
      class="home__loading"
    >
      Loading...
    </div>
  </div>
</div>
`;
